landoscript/src/landoscript/actions/version_bump.py

```diff
@@ -0,0 +1,122 @@
+import logging
```

Comment:
I think an argument could also be made to share the actual file manipulation logic between landoscript and treescript/githubscript. But that can be a future cleanup (and probably won't ever be a priority)

The actions you're concerned with are all Gecko specific anyway, so there's not much point. But in the future, we could have e.g a "pre-commit" action that applies lint fixes, or a "commitizen" action that generates a changelog. That kind of stuff might benefit from being shared across both scripts.

Reply:
That's a fair point. I don't really mind moving that sort of thing to shared code. It's not terribly harmful to live in `scriptworker_client` (or perhaps a new module?) even if there's never any re-use?

I know we've talked about this a bit at points, but it would probably be good to write down our long term vision for lando/github/treescript going forward. If I recall correctly, we anticipate treescript going away entirely, and any lando-enabled repo operations will move to landoscript (which will include all current supported hg repos?), and the remainder will move to githubscript? (I say this mainly selfishly, because I often get myself confused between the three of them...) The answers here may influence whether or not it's worthwhile to move some code out.

Reply:
Good point, could be worth writing up a quick RFC. I think you should just leave it here for now though, no need to get distracted by that yet.